test/cmdlineTests/name_simplifier/output

```diff
@@ -87,7 +87,7 @@ object "C_56" {
                 if or(gt(newFreePtr, 0xffffffffffffffff), lt(newFreePtr, memPtr)) { invalid() }
                 mstore(64, newFreePtr)
             }
-            function convert_t_stringliteral_6490_to_t_string() -> converted
+            function convert_stringliteralfd228f7ef267f3b474dd6ef84bae434cf5546eee948e7ca26df3eda1927_to_string() -> converted
```

Comment:
Are the new rules not picking up the stringliteral shortener?

Reply:
Fixed.